src/main/java/org/opentripplanner/middleware/controllers/api/AbstractUserController.java

```diff
@@ -98,6 +106,45 @@ private U getUserFromRequest(Request req, Response res) {
         return user;
     }
 
+    /**
+     * HTTP endpoint to delete the {@link U} entity, if it exists, from an {@link RequestingUser} token
+     * available from a {@link Request} (this is the case for '/api/secure/' endpoints).
+     */
+    private boolean deleteUserFromRequest(Request req, Response res) {
+        RequestingUser profile = Auth0Connection.getUserFromRequest(req);
+        if (profile == null) {
+            logMessageAndHalt(req,
+                HttpStatus.NOT_FOUND_404,
+                String.format(NO_USER_WITH_AUTH0_ID_MESSAGE, null),
+                null);
```

Comment:
What formatting are you using? This and a few other lines wrap were I wouldn't expect.

Reply:
I copied stuff over from `getUserFromRequest`, so I guess the formatting comes from there... I agree that `req` should be on its own line.

Reply:
Refactored in https://github.com/ibi-group/otp-middleware/pull/213/commits/af8b9324ec3b3da4b4fab50380bb353795b734cc.